internal/pkg/service/stream/storage/level/local/diskreader/config.go

```diff
@@ -0,0 +1,23 @@
+package diskreader
+
+import (
```

Comment:
I updated also `diskreader` package, to match `diskwriter` package style, to reduce confusing.